packages/dcos-calico/extra/dcos-calico-node.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=DC/OS calico node: initializes a new calico node and runs calico components
+After=docker.service
+Requires=docker.service
+
+[Service]
+EnvironmentFile=/opt/mesosphere/etc/calico-node.env
+ExecStartPre=-/usr/bin/docker rm -f dcos-calico-node
```

Comment:
We use `dcos-calico-node` instead of the `calico-node` by intention for fear that `calico-node` used by official document of project calico may interfere with ours though the case is rare.